### PR TITLE
Merging Weaviate notebooks to main

### PR DESCRIPTION
**Pull request by colin-openai**
Worth one additional review to ensure main notebook runs end-to-end

### CODE REVIEW

**Comment by ted-at-openai** [comment]
Suggestion: in the notebook, instead of saying the file is "pretty big" say it's "~700 MB". I expect a term like "pretty big" means different things to different people, and it will be hard for them to know what to expect.

**Comment by sebawita** [comment]
Hey @ted-at-openai, thank you for the feedback.

I am OK with "pretty big" => "~700 MB" change.

Is this the only change you need from us?

**Comment by ted-at-openai** [comment]
Still running through the example notebook. Will drop comments as I have them. :)

**Comment by ted-at-openai** [comment]
> Hey @ted-at-openai, thank you for the feedback.
> 
> I am OK with "pretty big" => "~700 MB" change.
> 
> Is this the only change you need from us?

Still running through the example notebook. Will drop comments as I have them. :)

**Comment by ted-at-openai** [comment]
"The other vector database option" -> "Another vector database option" (I assume this was written when there were just two)

**Comment by ted-at-openai** [comment]
I tried running `docker-compose up -d` and immediately hit a blocker: `no configuration file provided: not found`

Probably something simple, but if I ran into it, others will too.

**Comment by ted-at-openai** [comment]
I think instead of "You can start Weaviate up locally by navigating to this directory" it should say the actual directory. Phrases like "this directory" can always be a bit ambiguous. Here I think it means the directory of the yml file, not the directory of the notebook I was reading (my original interpretation).

**Comment by ted-at-openai** [comment]
Next error I hit was `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?` Do we need clearer instructions here?

**Comment by ted-at-openai** [comment]
> Next error I hit was `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?` Do we need clearer instructions here?

Solution to this error was to start Docker first. Maybe rephrase previous instruction to "After starting Docker, ..."

**Comment by ted-at-openai** [comment]
(sorry for all the comments here instead of in-line. looks like I can only do in-line comments for the lines affected by the latest commits and not the earlier ones)

**Comment by ted-at-openai** [comment]
Otherwise, everything ran perfectly. Code and output look clean. This is fantastic!

**Review by ted-at-openai — APPROVED** [review]

**Comment by sebawita** [comment]
Hi @ted-at-openai 
Thank you for your feedback, your changes (you were 100% spot on) and merging the PR.
This looks amazing.

Any chance we could update the Docker link to - https://www.docker.com (instead of https://www.docker.com/pricing)?

**Comment by ted-at-openai** [comment]
Updated: https://github.com/openai/openai-cookbook/pull/129
